examples/plot_visualize_bad_epochs.py

```diff
@@ -83,36 +79,34 @@
 
     # Same `dev_head_t` for all runs so that we can concatenate them.
     epoch.info['dev_head_t'] = epochs[0].info['dev_head_t']
+
 epochs = mne.epochs.concatenate_epochs(epochs)
 
 ###############################################################################
 # Now, we apply autoreject
 
-###############################################################################
-from autoreject import LocalAutoRejectCV, compute_thresholds
-from functools import partial
+from autoreject import LocalAutoRejectCV, compute_thresholds  # noqa
+from functools import partial  # noqa
 
 this_epoch = epochs['famous']
 thresh_func = partial(compute_thresholds, random_state=42)
 
-ar = LocalAutoRejectCV(thresh_func=thresh_func, verbose='tqdm')
+ar = LocalAutoRejectCV(thresh_func=thresh_func, verbose='progressbar')
```

Comment:
arff ... now I remember why I had tqdm here. There is something buggy about progressbar which is why `circle` fails on `master`. No output for more than 10 minutes ...

And circle isn't showing up on PRs any more for some reason.